rpc/transport/nats/nats_transport.go

```diff
@@ -78,7 +78,7 @@ func (c *natsTransportClient) Request(data []byte) ([]byte, error) {
 	return msg.Data, err
 }
 
-func (c *natsTransportServer) RegisterRequestHandler(handler func([]byte) []byte) error {
+func (c *natsTransportServer) RegisterRequestHandler(apiVersion string, handler func([]byte) []byte) error {
```

Comment:
Should the apiVersion be added to the `nitroRequestTopic`, similar to how the version is part of the url for websockets?
```golang
	nitroRequestTopic      = "nitro-request/api/v1"
```

Reply:
Went with the topic string format shown above